### PR TITLE
Allow using TLS cert credentials for admin account (autofund feature)

### PR DESCRIPTION
**Pull request by lumberj**
Note: autofunding only used for testing purposes

### CODE REVIEW

File: src/lib/config.js [hunk 1]
```diff
@@ -47,6 +47,22 @@ function parseCredentials () {
   }, {})
 }
 
+function parseAdminEnv () {
+  const adminUser = Config.getEnv(envPrefix, 'ADMIN_USER') || 'admin'
```
Comment (gip) [inline src/lib/config.js]: Why do we need to default to `admin`? A real, non-default user name should always be specified I'd think.

Comment (gip) [inline src/lib/config.js]: LGTM

Comment (lumberj) [inline src/lib/config.js]: This is existing behavior

Comment (lumberj) [inline src/lib/config.js]: Since this is only for debug / testing, it shouldn't matter

**Pull request merged**